generators/server/templates/src/test/java/package/web/rest/_LogsResourceIntTest.java

```diff
@@ -19,10 +19,11 @@
 package <%= packageName %>.web.rest;
 
 <%_ if (databaseType === 'cassandra') { _%>
-import <%= packageName %>.AbstractCassandraTest;
+import <%=packageName%>.AbstractCassandraTest;
 <%_ } _%>
-import <%= packageName %>.<%= mainClass %>;
-import <%= packageName %>.web.rest.vm.LoggerVM;
+import <%=packageName%>.<%= mainClass %>;<% if (authenticationType == 'uaa') { %>
```

Comment:
I think we have an un written rule 😜, and we even started changing templates to follow this sometime back. Hence i think its better to follow the same